go/felix/calc/policy_sorter_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package calc_test
+
+import (
+	. "github.com/projectcalico/felix/go/felix/calc"
+
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/projectcalico/libcalico-go/lib/backend/model"
+)
+
+var (
+	tenPointFive = 10.5
+)
+
+var _ = FDescribeTable("PolKV should stringify correctly",
```

Comment:
If I understand correctly, you need to remove the leading 'F' here (as it causes the 'Detected Programmatic Focus' status, which our CI thinks is a failure), and then retest.